<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaArchivos</class>
 <widget class="QMainWindow" name="ventanaArchivos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Administrador de archivos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="botonAtras">
        <property name="text">
         <string>Atr√°s</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="botonHome">
        <property name="text">
         <string>Home</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="botonRefresco">
        <property name="text">
         <string>Refrescar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboUnidad"/>
      </item>
      <item>
       <widget class="QLineEdit" name="rutaTexto">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="botonIr">
        <property name="text">
         <string>ir</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Carpetas</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="directoriosLista"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Archivos</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="archivosLista"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="botonCopiar">
            <property name="text">
             <string>Copiar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botonMover">
            <property name="text">
             <string>Mover</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bontonCarpeta">
            <property name="text">
             <string>Crear carpeta</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botonRenombrar">
            <property name="text">
             <string>Renombrar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="botonDescargar">
            <property name="text">
             <string>Descargar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botonSubir">
            <property name="text">
             <string>Subir</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botonBorrar">
            <property name="text">
             <string>Borrar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botonEjecutar">
            <property name="text">
             <string>Ejecutar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="progresoTransferencia">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
